### PR TITLE
Ability to drag and drop images onto game edit dialog media tab

### LINKED ISSUE #1156: Drag'n'Drop files to media tab in game edit dialog

### PR DESCRIPTION
**Pull request by Bobsans**
Fixes #1156 

In the presence of a large collection of games it is very convenient to simply drag and drop pictures, instead of opening dialog and searching for the desired picture.

### CODE REVIEW

**Comment by JosefNemec** [comment]
Could you please base this on `devel`? There are lot of refactoring changes for upcoming 5.0 release and I think there might be merge conflicts.

**Comment by Bobsans** [comment]
I fixed it for current version.

File: source/Playnite.DesktopApp/ViewModels/GameEditViewModel.cs [hunk 1]
```diff
@@ -2332,9 +2348,13 @@ public void UseExeIcon()
             EditingGame.Icon = icon;
         }
 
-        public void SelectCover()
+        public void SelectCover(string path = null) 
         {
-            var path = dialogs.SelectImagefile();
+            if (path == null)
```
Comment (JosefNemec) [inline source/Playnite.DesktopApp/ViewModels/GameEditViewModel.cs]: Should be checked via `IsNullOrEmpty`.

File: source/Playnite.DesktopApp/ViewModels/GameEditViewModel.cs [hunk 2]
```diff
@@ -2346,9 +2366,31 @@ public void SelectCover()
             }
         }
 
-        public void SelectBackground()
+        public void DropCover(DragEventArgs args) 
         {
-            var path = dialogs.SelectImagefile();
+            if (args.Data.GetDataPresent(DataFormats.FileDrop))
```
Comment (JosefNemec) [inline source/Playnite.DesktopApp/ViewModels/GameEditViewModel.cs]: Can you create a single unction for this? Seems like unnecessary code duplication for just one different line.

**Comment by darklinkpower** [comment]
@Bobsans just wondering if there has been progress in this feature? Looks like a pretty useful feature that would help a lot

File: source/Playnite.DesktopApp/ViewModels/GameEditViewModel.cs (hunk 1 above)
Comment (Bobsans) [reply source/Playnite.DesktopApp/ViewModels/GameEditViewModel.cs]: Why? This variable may be or null or path. It not be empty anycase.

**Pull request merged**